layouts/partials/footer.html

```diff
@@ -3,19 +3,25 @@
 <footer class="footer footer-light footer-bar">
     <div class="footer-py-30">
         <div class="container text-center">
-            <div class="row align-items-center">
-                    <div class="text-sm-start">
-                        <p class="mb-0 d-flex justify-content-end">
-                            {{ range .Site.Params.footer }}
-                                <a href="{{ relLangURL .url }}">{{ .name }}</a>
-                            {{ end }}                       
-                        </p>
-                    </div>
-                <div class="col-sm-6 mt-4 mt-sm-0 pt-2 pt-sm-0">
-                    <!-- <p>Privacy</p> -->
-                </div><!--end col-->
-            </div><!--end row-->
-        </div><!--end container-->
+            <ul class="nav justify-content-end">
+                <li class="nav-item">
+                    {{ $Site := .Site }}
+                    {{ with .File }}
+                    <a class="px-2" href="{{ $Site.Params.editURL }}{{ replace .Dir "\\" "/" }}{{ .LogicalName }}" target="blank">
+                        {{"Edit this page"}}
+                    </a>
```

Comment:
the onlick is dirts and might not work with opening in a new tab or with javascript deactivated.

Wouldn't a `rel=”nofollow”` suffice?

Reply:
> the onlick is dirts and might not work with opening in a new tab or with javascript deactivated.

Without JavaScript the Freifunk website does not fully work either (e.g. language switcher) and editing at GitHub also requires JavaScript. Without it the content and edit form is not even visible.

> Wouldn't a rel=”nofollow” suffice?

The idea behind masking the link is to reduce the number of outgoing links and prevent PageRank from beeing assigned to those links so the remaining internal links will be stronger. `nofollow` does not do that. With `nofollow` there will still be PageRank assigned to the links but the PageRank will not be passed on to the target pages. Compared to masking the links `nofollow` would therefore weaken the site.

We can however just use JavaScript to create the link so it will only become visible when it would also be usable:

```
<span id="edit-page-link" class="px-2" 
      data-url="{{ $Site.Params.editURL }}{{ replace .Dir "\\" "/" }}{{ .LogicalName }}">
</span>
<script>
document.addEventListener("DOMContentLoaded", function() {
    var editLink = document.getElementById('edit-page-link');
    if (editLink) {
        var url = editLink.getAttribute('data-url');
        editLink.innerText = "Edit this page";
        editLink.onclick = function() {
            window.open(url, '_blank');
        };
    }
});
</script>
```